### DISCUSSION TITLE
MacOS 10.1 Puma stuck on Happy Mac

### DISCUSSION THREAD

**Discussion by ThatOneWindowsFan** [utmapp/UTM#5644] [macOS]
**Describe the issue**  
Today, I tried to install MacOS 10.1 Puma on UTM with PowerPC Chip. After the first portion of setup (before the intro video), I'm stuck on the Happy Mac screen, with the beach ball spinning in the corner.

**Configuration**  
* UTM Version: 4.3.5
* macOS Version: Ventura 13.5
* Mac Chip (Intel, M1, ...): M1

**Upload VM**  
I'm not sure where the VM is, so please tell me.

**Comment by thehappylemon1** [comment]
Any updates on this issue? I also have it but on a real iBook g3 lol
